### PR TITLE
Add Prettier to linting

### LINKED ISSUE #316: Enable prettier for the `amp-github-apps` repo
For consistency, let's use the same `.prettierrc` as `amphtml`.

### PR DESCRIPTION
**Pull request by rcebulko**
Uses the .prettierrc from ampproject/amphtml

Closes #316

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 */node_modules/*
-bundle-size/*.example.js
-test-status/*.example.js
+bundle-size/*.example.js
```
Comment (danielrozenberg) [inline .eslintignore]: Superfluous whitespace

File: .eslintignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
 */node_modules/*
-bundle-size/*.example.js
-test-status/*.example.js
+bundle-size/*.example.js  
+test-status/*.example.js
```
Comment (danielrozenberg) [inline .eslintignore]: Pleasy change your editor's config to add a newline at the end of files >____<

Comment (rcebulko) [reply .eslintignore]: It used to autoformat and somehow I broke that =__= I'll get it figured out

File: .eslintignore (hunk 1 above)
Comment (rcebulko) [reply .eslintignore]: Done

**Pull request merged**